core/src/test/java/org/wildfly/extras/sunstone/tests/AbstractCloudProviderTest.java

```diff
@@ -155,19 +157,28 @@ public final void node() throws Exception {
                 // OK
             }
 
-            try (Node node = cloudProvider.createNode("mynode")) {
-                assertEquals(1, cloudProvider.getNodes().size());
-                assertNotNull(cloudProvider.getNode("mynode"));
+            try (CreatedNodes createdNodes = cloudProvider.createNodes("mynode", "othernode")) {
```

Comment:
This was originally to test the behavior of the cloud provider when nodes are incrementally being started and stopped. I'd like to keep that part of the code covered, but I understand that performance is a big concern.

How about we only do this test with one node (the "othernode" is really not relevant here and can be removed), and use some kind of mocking for testing how the cloud provider behaves when nodes are started and stopped?